src/main/java/htsjdk/tribble/annotation/Strand.java

```diff
@@ -24,29 +24,116 @@
 package htsjdk.tribble.annotation;
 
 /**
- * Enum for strand, which can be encoded as string
+ * Enum for strand, which can be encoded as a string
  */
 public enum Strand {
-    POSITIVE("+"), NEGATIVE("-"), NONE("!");  // not really sure what we should do for the NONE Enum
 
     /**
-     * How we represent the strand information as text
+     * Represents the positive or forward strand.
      */
-    private String encoding;
-    Strand(String str) {
-        encoding = str;
+    POSITIVE('+'),
+
+    /**
+     * Represents the negative or reverse strand.
+     */
+    NEGATIVE('-'),
+
+    /**
+     * Denotes that a strand designation is not applicable
+     * or is unknown.
+     */
+    NONE('!');  // not really sure what we should do for the NONE Enum
```

Comment:
I prefer the ```Optional<X>``` to handle that for those that really need to make such a distinction. The fact that there is as many non-proper enum constants (?, .) than proper one (+, -) is already smelly IMO. Anyone else can chime in? Can be de tie breaker?

Reply:
That's a change in the codecs, not in the enum,but I do not block the PR due to this discussion (I can propose a change in a following PR, either for the codec or for the enum). I am fine if only one "none" concept exists as far as it is encoded with the default for BED, because it is the codec which uses this currently (and also it is common in other formats).

For maintain backwards compatibility I guess that the best approach will be to deprecate `NONE` (to keep the `'!'` econding) in favor of a `MISSING`or `UNKNOWN` encoded with `'.'`. Once the `NONE` enum is removed, an alias can be added (similarly to the `POSITIVE`/`FORWARD` alias).

Reply:
I would go with "."

Reply:
where did the "!" suggestion come from?

Reply:
The `!` was the previous implementation... I guess that it comes from times when undefined wasn't `.` yet.

Reply:
oh, OK. leave as is then!

Reply:
I would like to have `.`, because it is actually the representation for the BED file (at the moment, the only Strand usage within the library).

Reply:
Unless someone have show that '!' is actually a used in practice I'm inclined to change it to '.'

Reply:
I'm not going to add yet another "NONE" constant in this pull request.  @magicDGS if you fill strong about this I suggest that you work on another pull request with that change. Hopefully you can show an example how that that change improve some other piece of code in htsjdk.

Reply:
I was referring in the past comment to do exactly the change from '!' to '.', as you've already did. Thanks!